### ISSUE TITLE
Background and typography global colors break in site settings editor

### ISSUE THREAD

**Issue by wwdes** [elementor/elementor#13057]
## Prerequisites

- [x] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [x] The issue still exists against the latest stable version of Elementor.

## Description
Any global colors set for background and typography will break while making changes in Site Settings mode. This has no effect on frontend. Reloading editor fixes the colors.

## Steps to reproduce
Basic patern:
Make changes in Site Settings, hit Update, make another change in Site Settings.

Example:
`Site Settings > Background` set to a global color
`Site Settings > Custom CSS` make some changes
Hit `Update` button
Make another change in in `Custom CSS`

100% consistent.

## Isolating the problem

- [x] This bug happens with only Elementor plugin active (and Elementor Pro).
- [x] This bug happens with a default WordPress theme active.
- [x] I can reproduce this bug consistently using the steps above.

## Environment

<details>
<summary>System Info</summary>
```
<!-- For bug reports, let us know about your system environment: Copy and paste the system info report from Elementor => System info, and paste it here or in http://pastebin.com/ -->
```
</details>

**Comment by MichaelDarko** [comment]
It's probably related to this: #12728

If you make change -> save -> make another change, some global colors break in the editor.

**Comment by DorShahar** [comment]
Merged issue

I merged this issue with  #12728, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.
Thanks for your feedback!
